src/pretix/control/forms/__init__.py

```diff
@@ -131,3 +130,37 @@ def __init__(self, attrs=None, date_format=None, time_format=None):
         )
         # Skip one hierarchy level
         forms.MultiWidget.__init__(self, widgets, attrs)
+
+class DatePickerWidget(forms.DateInput):
+
+    def __init__(self, attrs=None, date_format=None):
+        attrs = attrs or {}
+        if 'placeholder' in attrs:
+            del attrs['placeholder']
+        date_attrs = dict(attrs)
+        date_attrs.setdefault('class', 'form-control')
+        date_attrs['class'] += ' datepickerfield'
+
+        df = date_format or get_format('DATE_INPUT_FORMATS')[0]
+        date_attrs['placeholder'] = now().replace(
+            year=2000, month=1, day=1, hour=0, minute=0, second=0, microsecond=0
```

Comment:
What about using `month=12 ,day=31` in these placeholders to avoid confusion between month and day?

Reply:
> What about using month=12 ,day=31 in these placeholders to avoid confusion between month and day?

Absolutely, great idea!

Reply:
Maybe we could also clearly show that it's 24hrs (no am/pm), but I think also using `23:59:59` would be visually distracting. Using something like `18:00:00` seems a bit arbitrary. What do you think?

Reply:
I think 18 is fine, the year 2000 is just as arbitrary ;)